### PR TITLE
Add more fields to attendee CSV.

### PR DESCRIPTION
**Pull request by magdapoppins**
Almost done but some help still needed:

Try to solve https://github.com/codebar/planner/issues/922. Got the newbie status and student learning goals showing. Wasn't able to see if the coach skill list shows since I couldn't get myself added as a coach for the event in the dev environment. 

Also need some support with getting the tests passing again, now `workshop_presenter_spec.rb` has a failing test and I'm not sure why.

### CODE REVIEW

File: app/presenters/workshop_presenter.rb [hunk 1]
```diff
@@ -77,13 +77,26 @@ def attending_organisers
   end
 
   def attendee_array
```
Comment (notapatch) [inline app/presenters/workshop_presenter.rb]: The original attendee_array function was complicated. I wouldn't be confident refactoring it until I added more tests to see what it's up to. As it stands to add a column to the CSV you have to change the array of items in 3 places! Two directly in attended_array and one in a called sub-method attending_organisers.

```
  def attending_organisers
    organisers.map do |o|
      [o.full_name, 'ORGANISER']
    end
  end

```
Clearly, this isn't a user friendly function and the lack of dryness means if someone "just" wants to add another column there's a good chance of mistakes.

**Comment by notapatch** [comment]
Hi If you are looking at this over the weekend and I can be of any assistance - just ask.

**Comment by magdapoppins** [comment]
> Hi! If you are looking at this over the weekend and I can be of any assistance - just ask.

Hi! Are you in the codebar slack? I'm still having some trouble with the tests, and also with signing in as an admin locally to be able to see what's going on...

**Comment by notapatch** [comment]
Upgrading to admin has been simplified by a pull request this week. So fetch the latest code if you haven't already check you have a `bin/dadmin` and try it out. I have taken the quote from the README if it doesn't make sense ... then that's a bug.

I can see #codebar-website  and have said "hello world" ... I'll be on in an hour.  

-------------
#README  *snip*


5. Start the app

Run bin/dstart to start the app.

This should run a process which starts a server in a Docker container on your computer. This process won't finish - you'll know the server is ready when it stops doing anything and displays a message like this:

Rails 4.2.11 application starting in development on http://0.0.0.0:3000

(Optional) Note that to be able to use the page as an admin, you must first give yourself admin privileges. Make sure you have started your app and signed up as an user on your locally running app. Then run a script bin/dadmin <your email>.

**Comment by matyikriszta** [comment]
@magdapoppins once you have made yourself an admin you can RSVP yourself to any workshop as a coach from the admin workshop view page.

**Comment by notapatch** [comment]
@magdapoppins - how's it going with the PR? Since your message I've been hanging around slack. If I can be of any assistance (@not-a-patch - I should remove the hyphens) I'd be happy to help.

We have quite a few PR's open so if you aren't able to continue or would prefer to do something else then let me know and we can recycle the issue.

**Comment by despo** [comment]
Closing as it has been resolved by #1225

**Pull request closed without merging**